scripts/gulp/tasks/vr-test.ts

```diff
@@ -40,8 +40,10 @@ task('screener:runner', cb => {
     deployUrl: process.env.DEPLOYURL,
   });
 
-  const screenerStates = require(paths.docsDist('screenerStates.json'));
-  screenerConfig.states = screenerStates;
+  if (process.env.IS_ARTIFACT_PRESENT) {
+    const screenerStates = require(paths.docsDist('screenerStates.json'));
+    screenerConfig.states = screenerStates;
```

Comment:
shouldn't we terminate here ?  arent those states necessary for run ?

Reply:
no the screener proxy should be called to skip the northstar check completely. The default storybook integration can generate the states directly from storybook, but it's not a feature we have capacity for in the northstar team.

The states can only exist as a part of the workflow artifact. If there is a bug where the test should be but there is no artifact that's a completely separate issue